template/startblog/bloger-profile.html

```diff
@@ -0,0 +1,49 @@
+{%extends 'startblog/base.html'%}
+{%load static%}
+{%block content%}
+
+
+<section>
+    <div class="container">
+        <h2>Blogs of {{user}} </h2>
+        <form method='post' action= "{% url 'startblog:bloger_profile' request.user.id  %}" >
+            {%csrf_token%}
+          
+            <input type="email"  name = 'sub' class="form-control" placeholder ="Your Mail Id"required>
+            <input type ="submit"  class="btn btn-danger" value = 'Subscribe'>
+         
+        </form>
+    </div>
+{%for b in blogs%}
+    <section id="about" >
+      <div class="container">
+        
+        <div class="row">
+            <span class="border ">
+          <div class="">    
+           
+          
+           <a href = "{%url 'startblog:view_comments' b.id%}">   <img src="{{b.coverpic.url}}" class="img-thumbnail" alt=""></a>
+    {%for photos in photos%}
+      {%if photos.blog == b%}
```

Comment:
Handle this using a queryset and property in blog. 
If there is 10000 blog and each blog has 10 photos each, this will take hours to render

Reply:
done